metals/src/main/scala/scala/meta/internal/metals/NewFilesProvider.scala

```diff
@@ -98,6 +101,16 @@ class NewFilesProvider(
       }
   }
 
+  private def getName(
+      kind: String,
+      name: Option[String]
+  ): Future[Option[String]] = {
+    name match {
+      case Some(_) => Future.successful(name)
+      case None => askForName(kind)
```

Comment:
Let's also maybe ask if the name is empty? I think that breaks in the current behaviour.

Reply:
what you mean? Empty string? I can add a test but can you explain why do you think it breaks the current functionality?

Reply:
Anyway, I've added handling blank values